docs/guides/distribution/sign-macos.md

```diff
@@ -154,7 +154,7 @@ The workflow pulls the secrets from GitHub and defines them as environment varia
 
 [tauri-apps/tauri#592]: https://github.com/tauri-apps/tauri/issues/592
 [apple developer program]: https://developer.apple.com/programs/
-[notarizing macOS software before distribution]: https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution
```

Comment:
`macOS` is the correct spelling of the OS.

Reply:
Yeah that is correct! 
This seems to be an error on my part.
My guess is that it got auto-formated when I copy-pasted the text